Documentation/Books/AQL/Views/ArangoSearch/README.md

```diff
@@ -1,15 +1,173 @@
 ArangoSearch Views in AQL
 =========================
 
-Views of type **arangosearch** are an integration layer meant to seamlessly
+Views of type `arangosearch` are an integration layer meant to seamlessly
 integrate with and natively expose the full power of the
 [IResearch library](https://github.com/iresearch-toolkit/iresearch)
 to the ArangoDB user.
 
 They provide the capability to:
 * evaluate together documents located in different collections
 * search documents based on AQL boolean expressions and functions
-* sort the result set based on how closely each document matched the search 
+* sort the result set based on how closely each document matched the search
+
+Overview and Pitfalls
+---------------------
+
+Looking up documents in an ArangoSearch View is done via the `FOR` keyword:
+
+```js
+FOR doc IN someView
+  ...
+```
+
+`FOR` operations over ArangoSearch Views have an additional, optional, `SEARCH`
+keyword:
+
+```js
+FOR doc IN someView
+  SEARCH searchExpression
+```
+
+### `SEARCH`
+
+`SEARCH` expressions look a lot like `FILTER` operations, but have some noteable
+differences and pitfalls.
+
+First of all, filters and functions in `SEARCH`, when applied to documents
+_emitted from an ArangoSearch View_, work _only_ on attributes linked in the
+view! For example, given a collection `myCol` with the following documents:
```

Comment:
we have to mention `includeAllFields` option otherwise user may get wrong impression that we always require to specify attributes to index

Reply:
Fixed.